packages/mrc-ide-covidsim/src/imperial-params.ts

```diff
@@ -166,6 +168,17 @@ export function assignParameters(p: {}, input: input.ModelParameters) {
     }
   }
 
+  // Set all trigger incidences to 0 because we do not support adaptive triggers
+  Object.entries(p)
+    .filter(([key]) => key.includes('rigger incidence'))
```

Comment:
Make this easier to search for.
```suggestion
    .filter(([key]) => key.toLowerCase().includes('trigger incidence'))
```